app/models/node.rb

```diff
@@ -24,7 +24,7 @@ class Node < ActiveRecord::Base
   self.table_name = 'node'
   self.primary_key = 'nid'
 
-  def self.search(query:, order: :default, type: :natural, limit:)
+  def self.search(query:, order: :default, type: :natural, limit: "25")
```

Comment:
lets have this be an integer here, and convert to a string when splicing it in -- is that ok?

Reply:
Sure. Actually, I don't think we need to convert it to a string.